tianshou/trainer/base.py

```diff
@@ -19,6 +20,8 @@
     tqdm_config,
 )
 
+log = logging.getLogger(__name__)
```

Comment:
can we rename it to logger?
```suggestion
logger = logging.getLogger(__name__)
```

Reply:
See comment on respective suggestion below.